homeworks/rostyslav.havuka_RostyslavHavuka/11-dependency-injection/vehicle/src/app/pipes/car-link.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Car } from '../model';
+
+@Pipe({
+  name: 'carLink'
+})
+export class CarLinkPipe implements PipeTransform {
+
+  transform(car: Car): string {
+    return `https://www.google.com/search?q= + ${encodeURIComponent(car.vehicle)}`
```

Comment:
How does it work? 😄 
You concat not strings, you have one string with `+` inside